jellyfin_apiclient_python/api.py

```diff
@@ -118,6 +118,32 @@ def users(self, handler="", action="GET", params=None, json=None):
         else:
             return self._get("Users/{UserId}%s" % handler, params)
 
+    def media_folders(self, handler="", params=None, json=None):
+        return self._get("Library/MediaFolders/", params)
+
+    def virtual_folders(self, handler="", action="GET", params=None, json=None):
+        if action == "POST":
+            return self._post("Library/VirtualFolders", json, params)
+        elif action == "DELETE":
+            return self._delete("Library/VirtualFolders", params)
+        else:
+            return self._get("Library/VirtualFolders", params)
+
+    def physical_paths(self, handler="", params=None, json=None):
+        return self._get("Library/PhysicalPaths/", params)
+
+    def folder_contents(self, abspath="/", params={}, json=None):
+        params['path'] = abspath
+        params['includeFiles'] = params.get('includeFiles', True)
+        params['includeDirectories'] = params.get('includeDirectories', True)
+        return self._get("Environment/DirectoryContents", params)
```

Comment:
![image](https://github.com/jellyfin/jellyfin-apiclient-python/assets/8078788/dffc38e5-fa78-4037-b1bf-6a9dba22e483)
It is best practice not to mutate default arguments.

Reply:
Oh, that's nefarious. Because this isn't my code I didn't catch that.

It's probably best to not have mutable default arguments at all. (horror story: I almost exfiltrated a bunch of environment variable secrets when I was building read-the-docs for one of my projects because I has `os.environ` as a default param :fearful:)

There are few other places where this is a problem in the code, but that should be address in a separate PR.

This and other review concerns should be fixed now.